Cargo.toml

```diff
@@ -27,7 +27,9 @@ serde_derive = "1.0"
 regex = "1"
 lazy_static = "1.4"
 typed-arena = "2.0.2"
-reqwest = { version = "0.11.12", features = ["blocking"] }
+thiserror = "1.0.49"
+ureq = "2.8.0"
+anyhow = "1.0.75"
```

Comment:
The `thiserror` and `anyhow` crates are by the same author, who is also the author of the `serde` crate.  The `anyhow` crate is for application-level error handling, and the `thiserror` crate is for library-level error handling.